src/coord/hybrid/dist_hybrid_plan.h

```diff
@@ -22,10 +22,10 @@ extern "C" {
  * Builds the static portion of the distributed pipeline
  * @param hreq the hybrid request
  * @param hybridParams pipeline parameters needed for building the pipeline
- * @param[out] us upstream parameters
+ * @param[out] lookups array to populate with lookups for each subquery
  * @param status if there is an error
  */
-int HybridRequest_BuildDistributedPipeline(HybridRequest *hreq, HybridPipelineParams *hybridParams, AREQDIST_UpstreamInfo *us, QueryError *status);
+arrayof(char*) HybridRequest_BuildDistributedPipeline(HybridRequest *hreq, HybridPipelineParams *hybridParams, RLookup **lookups, QueryError *status);
```

Comment:
can we add a documentation about the return type and say who is responsible for releasing the returned value?

If it is the caller, then I would add as argument perhaps